lib/tasks/rummager.rake

```diff
@@ -1,4 +1,9 @@
 namespace :rummager do
+  desc "Re-index one Document. Takes a CONTENT_ID as argument."
+  task resend_document: :environment do
+    Document.find_by(content_id: ENV.fetch("CONTENT_ID")).published_edition.update_in_search_index
```

Comment:
I think using using [rake arguments](https://robots.thoughtbot.com/how-to-use-arguments-in-a-rake-task) to pass in the `content_id` would be a bit more idiomatic for a rake task, and should't make too big a difference to the Jenkins invocation.